### ISSUE TITLE
Error text: OK (rejected) It seems that the connection to server has been lost. Please check your network connectivity and server status.

### ISSUE THREAD

**Issue by alexissel** [phpmyadmin/phpmyadmin#18049]
### Describe the bug

When `auth_type = cookie` the log in doesn't work and also not showing any error messages.
When `auth_type = http` the log in works but it shows the following message:

```
Error in processing request
Error code: 200
Error text: OK (rejected)
It seems that the connection to server has been lost. Please check your network connectivity and server status.
```

### Expected behavior

To successfully log into PMA.

### Server configuration

- Operating system: AlmaLinux 9.2
- Web server: Apache 2.4.53
- Database version: MariaDB 10.5.16
- PHP version: 8.0.20
- phpMyAdmin version: 5.2.0

### Client configuration

- Browser: Chrome
- Operating system: MacOS Monterey v12.6.3

**Comment by williamdes** [comment]
Can you confirm that the snapshot version also does not work?
What is your apache2 configuration?

If cookie login does not work there is something to dig about cookies, are you using a proxy?

**Comment by alexissel** [comment]
@williamdes, any news about this?

**Comment by zpaul91** [comment]
Hi, 
I found a solution, 
You need to disable version-check, in config.inc.php 
`$cfg['VersionCheck'] = false;`

I found that this route returns nothing, so that's not parsing as json -> error. 
Disabling it, you haven't the message anymore.

**Comment by alexissel** [comment]
I don't know exactly what caused this issue but I think it was due to a wrong configuration.
I had to reinstall the modules to get rid of the problem.

**Comment by alexissel** [comment]
The server modules (apache, php, mysql)

**Comment by joho1968** [comment]
> I had same issue, until I found this thread. Disable version check works. I then looked at the response from the version check XHR call, it showed an PHP error. And it turned out that curl_exec() function was disabled in php.ini

Perhaps it'd be easier to find this issue if phpMyAdmin did some basic `if ( ! function_exists('whatever_function_is_required') )` ... :thinking:

**Comment by williamdes** [comment]
Fixed by 69f9172a4c8f691dc0d9d993df73709487cf13e7
